lib/languages/js/helpers/index.js

```diff
@@ -38,7 +38,7 @@ helpers.setParam = function(param) {
     case 'body':
       return `if ('undefined' !== typeof params.${param.name}) { req.data = params.${param.name}; }`;
     case 'multipart':
-      return `if ('undefined' !== typeof params.${param.name}) { req.data.${param.name} = params.${param.name}; }`;
+      return `if ('undefined' !== typeof params.${param.name}) { req.data.${param.name} = ${param.type === 'object' ? `JSON.stringify(params.${param.name})` : `params.${param.name}`}; }`;
```

Comment:
this logic is mostly stolen from the query case above, all creative work is derivative